extension/copy-on-y.js

```diff
@@ -0,0 +1,16 @@
+/* globals utils */
+
+'use strict';
+
+const Y_KEYCODE = 89;
+
+window.enableCopyOnY = () => {
+	$(document).on('keyup', ({keyCode}) => {
+		if (keyCode === Y_KEYCODE) {
+			const commitIsh = $('.commit-tease-sha').text().trim();
+			const uri = location.href.replace(/\/blob\/(.+)\//, `/blob/${commitIsh}/`);
```

Comment:
1. Pressing `y` causes a page refresh; at the time our code runs, the value in the URI is going to point to the branch, rather than the commit hash.

2. Yeah, way too open. `/\/blob\/(\w+)\//` won't work, though, because it'll fail when the branch name has a dash in it. `/\/blob\/[\w-]+\//` would probably work. I'll push an update a bit later tonight.